api/peripheral_api/routes/Printer.js

```diff
@@ -23,7 +22,10 @@ router.get('/healthCheck', (req, res) => {
 
 const s3 = new AWS.S3({ apiVersion: '2012-11-05' });
 router.post('/sendPrintRequest', async (req, res) => {
-
+  /* eslint-disable-next-line */
+  if (s3BucketKeys.AWSACCESSKEYID == 'NOT_SET'&& s3BucketKeys.AWSSECRETKEY == 'NOT_SET') {
```

Comment:
you can make this if statement multi lines to avoid the lint error
if (condition1
  && condition2) {